cmd/security-secrets-setup/testdata/test-example-security.yml

```diff
@@ -0,0 +1,625 @@
+    image: edgexfoundry/docker-edgex-consul:1.1.1
+    ports:
+      - "8400:8400"
+      - "8500:8500"
+    container_name: edgex-core-consul
+    hostname: edgex-core-consul
+    networks:
+      edgex-network:
+        aliases:
+            - edgex-core-consul
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+      - consul-scripts:/consul/scripts
+      - vault-config:/vault/config
+      - /run/edgex/secrets:/run/edgex/secrets:ro    
+    depends_on:
+      - volume
+
+  config-seed:
+    image: nexus3.edgexfoundry.org:10004/docker-core-config-seed-go:1.1.0
+    container_name: edgex-config-seed
+    hostname: edgex-core-config-seed
+    networks:
+      edgex-network:
+        aliases:
+            - edgex-core-config-seed
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+    depends_on:
+      - volume
+      - consul
+
+  vault:
+    image: vault:1.0.3
+    container_name: edgex-vault
+    hostname: edgex-vault
+    networks:
+      edgex-network:
+        aliases:
+            - edgex-vault
+    ports:
+      - "8200:8200"
+    cap_add:
+      - "IPC_LOCK"
+    entrypoint: ["/vault/init/start_vault.sh"]
+    environment:
+      - VAULT_ADDR=https://edgex-vault:8200
+      - VAULT_LOCAL_CONFIG=${VAULT_LOCAL_CONFIG}
+      - VAULT_CONFIG_DIR=/vault/config
+      - VAULT_UI=true
+    volumes:
+      - vault-file:/vault/file
+      - vault-logs:/vault/logs
+      - vault-init:/vault/init:ro
+      - /run/edgex/secrets/edgex-vault:/run/edgex/secrets/edgex-vault:ro
+    depends_on:
+      - security-secrets-setup
+      - volume
+      - consul
+
+  security-secrets-setup:
+    image: edgexfoundry/docker-edgex-secrets-setup-go:1.1.1-dev
+    container_name: edgex-secrets-setup
+    hostname: edgex-secrets-setup
+    tmpfs:
+      - /run
+    command: "generate"
+    #command: "cache"
+    #command: "import"
+    volumes:
+      - secrets-setup-cache:/etc/edgex/pki
+      - vault-init:/vault/init
+      - /run/edgex/secrets:/run/edgex/secrets
+    depends_on:
+      - volume
+
+  vault-worker:
+    #image: nexus3.edgexfoundry.org:10004/docker-edgex-security-secretstore-setup-go:1.1.0
+    image: edgexfoundry/docker-edgex-security-secretstore-setup-go:1.1.1-dev
+    container_name: edgex-vault-worker
+    hostname: edgex-vault-worker
+    entrypoint: >
+      /bin/sh -c 
+      "until /consul/scripts/consul-svc-healthy.sh security-secrets-setup; do sleep 1; done;
+      /security-secretstore-setup --init=true --vaultInterval=10"
+    networks:
+      edgex-network:
+        aliases:
+            - edgex-vault-worker
+    volumes:
+      - vault-config:/vault/config
+      - consul-scripts:/consul/scripts
+      - /run/edgex/secrets/ca/ca.pem:/run/edgex/secrets/ca/ca.pem:ro
+      - /run/edgex/secrets/edgex-vault:/run/edgex/secrets/edgex-vault:ro
```

Comment:
Remove `- /run/edgex/secrets/edgex-vault:/run/edgex/secrets/edgex-vault:ro`

/run -> /tmp for the rest.